.changeset/two-houses-work.md

```diff
@@ -0,0 +1,5 @@
+---
+"@eth-optimism/contracts": patch
+---
+
+Updates the `yarn build` command to not error
```

Comment:
I think we do not need a changeset since `yarn build` is only needed locally? Perhaps to bump the Docker image's construction? If so that makes sense.